### PR TITLE
feat: KPI-21 Create the demo for the frontend project

### PR DESCRIPTION
**Pull request by ridouku**
## Jira Link

[Go to Jira](https://kpinetwork.atlassian.net/browse/KPI-21)

## Types of changes

- [ ] Hotfix (Bug detected)
- [x] Feature (Issue from an Active Sprint)

## Checklist:

- [ ] Unit Test.
- [ ] Coverage.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -50,3 +50,71 @@ debug.log
...
+*.bak
+
+# Ignored Terraform files
+*gitignore*.tf
+
+# Ignore Mac .DS_Store files
+.DS_Store
+
+# Ignored vscode files
+.vscode/
```
Comment (kellycastrof) [inline .gitignore:103]: Files like .env can be added to the gitignore

File: README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+
+If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+
+Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+
+You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+
+## Learn More
```
Comment (kellycastrof) [inline README.md:42]: I don't think this part is necessary, but it can be left now as it is a demo.

Comment (ridouku) [reply README.md:42]: I left this part because in the future everything inside of README.md going to change

**Review by kellycastrof — APPROVED** [review]
👍

**Pull request merged**